starter-code/pizza.js

```diff
@@ -1 +1,64 @@
 // Write your Pizza Builder JavaScript in this file.
+
+'use strict'
+
+function main(){
+
+    var sauce = '$0 regular sauce';
+    var crust = '$0 regular crust';
+
+    var $peppers = $('.green-pepper');
+    var $peppersButton = $('.btn-green-peppers');
+    var $pepperList = $('.price li:nth-child(1)');
+        $peppersButton.click(function(){
+            $peppersButton.toggleClass('active')
+            $peppers.toggle(function(){
+            });
+        });
+
+    var $pepperoni = $('.pep');
+    var $pepButton = $('.btn-pepperonni');
+        $pepButton.click(function(){
+            $pepButton.toggleClass('active')
+            $pepperoni.toggle(function(){
+            });
+        });
+
+    var $mushrooms = $('.mushroom');
+    var $mushButton = $('.btn-mushrooms');
+        $mushButton.click(function(){
```

Comment:
Do you see how you repeat the same code again and again. Maybe you can abstract that into a function.